packages/ra-ui-materialui/src/input/ArrayInput/ArrayInput.stories.tsx

```diff
@@ -392,11 +452,113 @@ export const NestedInline = () => (
+                                                    source="content"
+                                                    helperText={false}
+                                                    sx={{ width: 200 }}
+                                                />
+                                            </TranslatableInputs>
+                                        </SimpleFormIterator>
+                                    </ArrayInput>
+                                </SimpleFormIterator>
+                            </ArrayInput>
+                        </SimpleForm>
+                    </Edit>
+                )}
+            />
+        </Admin>
+    </TestMemoryRouter>
+);
+
+export const NestedInlineNoTranslation = () => (
+    <TestMemoryRouter initialEntries={['/orders/1']}>
+        <Admin
+            dataProvider={
+                {
+                    getOne: () => Promise.resolve({ data: orderNested }),
+                    update: (_resource, params) => Promise.resolve(params),
+                } as any
+            }
+            i18nProvider={testI18nProvider()}
+        >
+            <Resource
+                name="orders"
+                edit={() => (
+                    <Edit
+                        mutationMode="pessimistic"
+                        mutationOptions={{
+                            onSuccess: data => {
+                                console.log(data);
+                            },
+                        }}
+                    >
+                        <SimpleForm>
+                            <TextInput source="customer" helperText={false} />
+                            <DateInput source="date" helperText={false} />
+                            <ArrayInput source="items">
+                                <SimpleFormIterator
+                                    inline
+                                    sx={{
+                                        '& .MuiStack-root': {
+                                            flexWrap: 'wrap',
+                                        },
+                                    }}
+                                >
+                                    <TranslatableInputs locales={['en', 'fr']}>
+                                        <Labeled source="name">
+                                            <TextField
+                                                source="name"
+                                                sx={{ width: 200 }}
+                                            />
+                                        </Labeled>
+                                    </TranslatableInputs>
+                                    <TranslatableFields locales={['en', 'fr']}>
+                                        <TextField
+                                            source="name"
+                                            sx={{ width: 200 }}
+                                        />
+                                        <TextField
```

Comment:
why is it doubled?

Reply:
Because when there's only one child, the `TranslatableFields` does not add labels

Reply:
A comment in the code will make future you aware of that peculiarity